core/src/main/java/org/pac4j/jax/rs/annotations/Pac4JCallback.java

```diff
@@ -57,4 +57,13 @@
      * @return value for {@link CallbackFilter#setMultiProfile(Boolean)}
      */
     boolean[] multiProfile() default {};
+
+    /**
+     * Note that this parameter only takes one value at most: empty array (default) is used to represent default pac4j
+     * setting, one boolean will be used by the filter, and more than one boolean will fail the resource method
+     * initialisation.
+     * 
+     * @return value for {@link CallbackFilter#setSaveInSession(Boolean)}
+     */
+    boolean[] saveInSession() default {};
```

Comment:
Why not using `Boolean`?

Reply:
Why would you want something that can have the value `null` here?

The point of those annotations is that they allow the user to express themselves without over thinking: if they put a value it is used, if they do not, the default is used.

Reply:
OK